source/guides/stacks/mers.rst

```diff
@@ -4,7 +4,8 @@ Media Reference Stack
 #####################
 
 The Media Reference Stack (MeRS) is a highly optimized software stack for
-Intel® architecture to enable media prioritized workloads, such as transcoding and analytics.
+Intel® Architecture Processors (CPU) and Intel® Processor Graphics (GPU) to
```

Comment:
CPU is not an abbr for Intel Architecture Processors

Reply:
I think you can drop the "(CPU)" and "(GPU)" from this sentence entirely. 

First use of CPU and GPU in the rest of the article, use :abbr:`CPU (Central Processing Unit)` and :abbr:`GPU (Graphics Processing Unit)`